packages/e2e/jest.config.js

```diff
@@ -1,22 +1,48 @@
-const project = (displayName) => ({
-  displayName,
+const commonProjectProps = {
   preset: 'ts-jest',
   setupFiles: ['dotenv/config'],
-  testMatch: [`<rootDir>/test/${displayName}/**/*.test.ts`],
   transform: { '^.+\\.test.ts?$': 'ts-jest' }
+};
+
+const project = (displayName) => ({
+  displayName,
+  testMatch: [`<rootDir>/test/${displayName}/**/*.test.ts`],
+  ...commonProjectProps
 });
 
+// jest tests suitable to run in an environment with real ADA (not tADA)
+const realAdaTestFileNames = [
+  'delegation',
+  'metadata',
+  'mint',
+  'multisignature',
+  'nft',
+  'puchDbWalletStores',
```

Comment:
```suggestion
  'pouchDbWalletStores',
```

Reply:
Great catch 🕵️ !

Reply:
If this is the only required change, I would like to force-push it to avoid the extra CI run and round of reviews needed when squashing.

Reply:
Yes, force-push is fine